### PR TITLE
Added support for more themes

### PR DESCRIPTION
**Pull request by eKeiran**
Most of the CSS stylesheets from the original repository are functional, only some that were too different in structure were omitted.

### CODE REVIEW

**Review by chriskthomas — CHANGES_REQUESTED** [review]
Hi,

This is a massive undertaking. I was not expecting so many themes in one go. Thank you! I made some comments on the php files. A few broad notes:

### Move/delete files

* Please put the `Templates` folder in the `src` folder and rename to `templates` (it is case sensitive outside of Windows).
* Delete the js files in every folder except in `PurpleSideOfTheForce`, which actually does something (this can be made to work later)
* Delete all of the index.html files

### PHP code considerations

* Please use globs to make the list of themes and check for the presence of the input in the array rather than the filesystem
* Use `/` for paths instead of `\`

### HTML/CSS considerations (within php files)

* Please avoid inline styles
* Style tags can only exist in the header
* Let's avoid making the html depend on specific themes. In this case (Pride) it is avoidable.

Thank you for the contribution. Overall, I think this is very ambitious and I can see that you have put a lot of work into testing this. Great job! If you would prefer that I wrap this up, just let me know.

**Comment by chriskthomas** [comment]
For moving the files btw, you can just do:

```sh
git mv Templates src/templates

# delete index.html files
rm src/templates/*/index.html

# temporarily move index.js from PurpleSideOfTheForce to avoid deletion
git mv src/templates/PurpleSideOfTheForce/index.js foo.js

# delete index.js files
rm src/templates/*/index.js

# Restore preserved js file
git mv foo.js src/templates/PurpleSideOfTheForce/index.js
```

File: src/index.php [hunk 1]
```diff
@@ -93,6 +94,53 @@
...
+              "Pink",
+              "Pride",
+              "PurpleSideOfTheForce",
+              "RedZone",
+              "Retro Game",
+              "Shades Of Purple",
+              "Sixties",
+              "Sky",
+              "Standard"
+            ];
```
[the remark this answers is not in the document]
Comment (eKeiran) [reply src/index.php:135]: Alright! I'll replace it with glob

File: src/api.php [hunk 2]
```diff
@@ -23,37 +23,68 @@
...
+
     <?php if (!empty($user_photo)) {?>
     <img id="userPhoto" src="<?=$user_photo?>" alt="User Photo">
     <?php }?>
 
     <a href="<?=(!empty($_POST["url"]) ? $_POST["url"] : ".")?>" id="userName"><?=$_POST["name"]?></a>
 
     <?php if (!empty($_POST["description"])) {?>
-    <p><?=$_POST["description"]?></p>
+        <div class="container"><p style="text-align: center; font-size: 1.25rem;"><?=$_POST["description"]?></p>  </div>
```
[the remark this answers is not in the document]
Comment (eKeiran) [reply src/api.php:75]: Ooh yes, cool!

File: src/api.php [hunk 3]
```diff
@@ -23,37 +23,68 @@
...
-    <style>
-        #userName,.link,p{display:block;font-family:var(--font);text-align:center;text-decoration:none;font-size:1.25rem}.link:hover,body{background-color:var(--bgColor)}:root{--bgColor:#1C1C1C;--accentColor:#E6E6E6;--font:-apple-system,BlinkMacSystemFont,avenir next,avenir,segoe ui,helvetica neue,helvetica,Cantarell,Ubuntu,roboto,noto,arial,sans-serif}#userPhoto{width:110px;height:110px;display:block;margin:35px auto 20px;border-radius:50%}p{color:#bbb;font-style:italic}#userName{color:#bbb;font-weight:600;line-height:1.25;width:100%}#links{max-width:675px;width:auto;display:block;margin:27px auto}.link{background-color:var(--accentColor);color:var(--bgColor);margin-bottom:20px;padding:17px;transition:.25s cubic-bezier(.08,.59,.29,.99);border:solid var(--accentColor) 2px}.link:hover{color:var(--accentColor)}
-    </style>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/fork-awesome@1.2.0/css/fork-awesome.min.css" integrity="sha256-XoaMnoYC5TH6/+ihMEnospgm0J1PM/nioxbOUdnM8HY=" crossorigin="anonymous">
+    <style>
+    <?php
+if (isset($_POST['theme'])) {
+    $selectedTheme = $_POST['theme'];
+    $themeFolderPath = '../Templates/' . $selectedTheme;
```
[the remark this answers is not in the document]
Comment (eKeiran) [reply src/api.php:38]: I see! Thanks for letting me know. Yep, I'll do it using glob soon

**Comment by eKeiran** [comment]
Completed the requested changes:

1. Moving/Deleting the requested files.
2. PHP: Using Globs to make an array of themes and validating input against the array for dealing with the security risk.
3. CSS/HTML: Removed inline styles and implemented body::before for the Pride theme.

Additional changes:
PHP:
1. Added favicon support.
2. Implemented audio functionality for the PurpleSideOfTheForce theme.

**Comment by chriskthomas** [comment]
Thanks! I don't think I can merge this directly. I think I'll just roll these changes into the repo manually and set you as the author of the commit

**Comment by chriskthomas** [comment]
Hi @eKeiran. I think it might be cleaner to put all the styles in a separate repo and keep the php here. I put your extensive css edits over here chriskthomas/linkfree-themes#2 in a separate repo.

I will then merge the php edits over here and make the css files available through a git submodule.

**Comment by chriskthomas** [comment]
Merged in the form of #13

**Pull request closed without merging**